src/main/java/by/custom/utilcalculator/service/MessagesManager.java

```diff
@@ -1,131 +1,133 @@
 package by.custom.utilcalculator.service;
 
-import by.custom.utilcalculator.repository.resources.Commands;
-import by.custom.utilcalculator.repository.steps.*;
+import by.custom.utilcalculator.directory.BotEntity;
+import by.custom.utilcalculator.directory.resources.Commands;
+import by.custom.utilcalculator.directory.steps.*;
 import org.telegram.telegrambots.meta.api.methods.send.SendMessage;
 import org.telegram.telegrambots.meta.api.objects.Message;
 
-public class MessagesCreator {
+public class MessagesManager {
```

Comment:
Now I think we have a lot of managers and `MessageCreator` reflects a sense better.

Reply:
Renamed it to MessagesCreator